libhirte/src/service/service.h

```diff
@@ -0,0 +1,6 @@
+#pragma once
+
+#define HIRTE_OBJECT_PATH "/org/containers/hirte"
+#define HIRTE_INTERFACE_BASE_NAME "org.containers.hirte"
+
+char *assemble_interface_name(char *name_postfix);
```

Comment:
Actually no, why? :sweat_smile:

Reply:
Shouldn't it be in include?

Reply:
Ah, sorry I misunderstood your question. 
Yes and no. I thought of `assemble_interface_name` more of an "internal interface", include would contain "public" ones. But you are right - since we don't really have a public API yet, I'll move it there.

Reply:
Moved :heavy_check_mark: